package.json

```diff
@@ -48,14 +48,14 @@
     "gulp-uglify": "^1.4.2",
     "gulp-wrapper": "^1.0.0",
     "jasmine-core": "^2.3.4",
-    "jasmine-spec-reporter": "^2.2.3",
-    "jscs": "^2.3.5",
+    "jasmine-spec-reporter": "^3.1.0",
+    "jscs": "^3.0.7",
     "jshint": "^2.8.0",
     "jshint-stylish": "^2.0.1",
-    "karma": "^0.13.14",
-    "karma-jasmine": "^0.3.5",
-    "karma-phantomjs-launcher": "^0.2.1",
-    "karma-spec-reporter": "^0.0.23",
-    "phantomjs": "^1.9.18"
+    "karma": "^1.3.0",
+    "karma-jasmine": "^1.1.0",
+    "karma-phantomjs-launcher": "^1.0.2",
+    "karma-spec-reporter": "^0.0.26",
+    "phantomjs-prebuilt": "^2.1.14"
```

Comment:
I have updated these in the `2.0.0` branch so you can remove these changes from `package.json` to avoid merge conflicts.